lib/rails_erd/domain.rb

```diff
@@ -135,6 +135,20 @@ def rails_models
         ActiveStorage::VariantRecord
         ActionText::RichText
         ActionText::EncryptedRichText
+        SolidQueue::Message
```

Comment:
I believe `SolidQueue::Message` is now `SolidCable::Message`. Maybe include both for backward compatibility?